corehq/apps/geospatial/views.py

```diff
@@ -227,13 +227,12 @@ def post(self, request, *args, **kwargs):
 @require_GET
 @login_and_domain_required
 def get_paginated_cases_or_users_without_gps(request, domain):
-    page = int(request.GET.get('page', 1))
-    limit = int(request.GET.get('limit', 5))
-    case_or_user = request.GET.get('data_type', 'case')
+    page = max(int(request.GET.get('page', 1)), 1)
+    limit = max(int(request.GET.get('limit', 5)), 1)
     query = request.GET.get('query', '')
-    assert page > 0, 'Page must be a positive number'
-    assert limit > 0, 'Limit must be a positive number'
-    assert case_or_user == 'case' or case_or_user == 'user', 'Data type must be "case" or "user"'
+    case_or_user = request.GET.get('data_type', 'case')
+    if case_or_user not in ['case', 'user']:
+        case_or_user = 'case'
```

Comment:
nit: just thinking if we should simply return empty response in such a case.

Reply:
I'm just thinking of scenarios where the user navigates to the page without setting the `data_type` URL param. In such a case, I feel defaulting to case might be better than the user simply seeing an empty table and wondering what went wrong.

We could have a warning on the page if this is not set, but that might just introduce unnecessary UI clutter.